src/dotnet/Chat.UI.Blazor/Pages/ChatPage.razor

```diff
@@ -104,6 +115,7 @@
     public record Model {
         public bool IsUnavailable { get; init; }
         public bool MustLogin { get; init; }
+        public bool IsRightPanelShowed { get; init; }
```

Comment:
I suggest to rename `IsRightPanelShowed -> IsRightPanelDisplayed`

Reply:
Done